src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -174,6 +174,9 @@
   <data name="IDS_FeatureStaticClasses" xml:space="preserve">
     <value>static classes</value>
   </data>
+  <data name="IDS_FeatureReadonlyStructs" xml:space="preserve">
```

Comment:
Nit: if we're going with ReadOnly capitalization, it would apply to IDS_... too.